.github/workflows/wheels-macos.yml

```diff
@@ -12,44 +12,29 @@ on:
 
 env:
   REPO_DIR: Pillow
-  BUILD_DEPENDS: ""
   TEST_DEPENDS: "pytest pytest-timeout"
-  WHEEL_SDIR: wheelhouse
 
 jobs:
   build:
-    name: ${{ matrix.os-name }} ${{ matrix.python }} ${{ matrix.platform }}
-    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.python }} ${{ matrix.platform }}
+    runs-on: "macos-latest"
     strategy:
       fail-fast: false
       matrix:
-        os: [ "macos-latest", "ubuntu-20.04" ]
-        python: [ "pypy3.7-7.3.8", "pypy3.8-7.3.8", "3.7", "3.8", "3.9", "3.10" ]
-        platform: [ "arm64", "x86_64", "i686" ]
+        python: [ "3.7", "3.8", "3.9", "3.10", "pypy3.7-7.3.8", "pypy3.8-7.3.8" ]
+        platform: [ "arm64", "x86_64" ]
```

Comment:
Let's move the slowest PyPys to the top, and for macOS put the slower x86_64 first:

![image](https://user-images.githubusercontent.com/1324225/160796400-ad1c8a7a-3277-4398-a97c-4f8e4da9dffa.png)


```suggestion
        python: [ "pypy3.7-7.3.8", "pypy3.8-7.3.8", "3.7", "3.8", "3.9", "3.10" ]
        platform: [ "x86_64", "arm64" ]
```